frontend/test/metabase/scenarios/native/native-mongo.cy.spec.js

```diff
@@ -20,9 +20,11 @@ describe("scenarios > question > native > mongo", () => {
   });
 
   it("can save a native MongoDB query", () => {
-    cy.get(".ace_content").type(`[ { $count: "Total" } ]`, {
-      parseSpecialCharSequences: false,
-    });
+    cy.get(".ace_content")
+      .first()
```

Comment:
I'm not sure why there are more than one `.ace_concent`, I suspect there is a bigger issue at play here.

Reply:
This should not be fixed for Mongo, but in general.
There's a `.NativeQueryEditor .ace_editor .ace_content`, which is the actual editor, then there's a `.ace_autocomplete .ace_content`, which is the autosuggest.

Reply:
Very good catch @flamber, thank you!

Reply:
Apparently this is a red herring. For a real fix (hopefully), see PR #21123.